docs/deployments/configuration.md

```diff
@@ -792,20 +797,21 @@ Path to the private key file.
 
 Allows specific ciphers to be set.
 
-`mlts` - _Type_: boolean | object; _Default_: false
-
-This can be configured to enable mTLS based authentication for incoming connections. If enabled with default options (by setting to `true`), the client certificate will be checked against the certificate authority specified with `certificateAuthority`. And if the certificate can be properly verified, the connection will authenticate users where the user's id/username is specified by the `CN` (common name) from the client certificate's `subject`, by default.
-
-You can also define specific mTLS options by specifying an object for mtls with the following (optional) properties which may be included:
+If you want to define multiple certificates that are applied based on the domain/host requested via SNI, you can define an array of TLS configurations. Each configuration can have the same properties as the root TLS configuration, but can (optionally) also have an additional `host` property to specify the domain/host that the certificate should be used for:
 
-`user` - _Type_: string; _Default_: Common Name
-
-This configures a specific username to authenticate as for mTLS connections. If a `user` is defined, any authorized mTLS connection (that authorizes against the certificate authority) will be authenticated as this user.
-This can also be set to `null`, which indicates that no authentication is performed based on the mTLS authorization. When combined with `required: true`, this can be used to enforce that users must have authorized mTLS _and_ provide credential-based authentication.
+```yaml
+tls:
+  - certificate: ~/hdb/keys/certificate1.pem
```

Comment:
I just tested this specific bit out. HarperDB did not like using `~`. I'm worried that there may be confusion in the docs around having to use the full path since they differ only slightly (`/home/harperdb` in container vs. `/home/ubuntu` on the host).

it looks like the relative path seems to work:
```
  - certificate: hdb/keys/certificate1.pem
```